packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -1173,12 +1180,15 @@ private void closeCaptureSession() {
 
   public void close() {
     Log.i(TAG, "close");
-    closeCaptureSession();
 
     if (cameraDevice != null) {
       cameraDevice.close();
       cameraDevice = null;
+      captureSession = null;
```

Comment:
can we add a comment that calling [CameraDevice#close](https://developer.android.com/reference/android/hardware/camera2/CameraDevice#close()) doesn't require closing the session, and it's faster as documented.